modules/src/core/ics02_client/events.rs

```diff
@@ -1,8 +1,8 @@
 //! Types for the IBC events emitted from Tendermint Websocket by the client module.
 
 use serde_derive::{Deserialize, Serialize};
-use tendermint::abci::tag::Tag;
-use tendermint::abci::Event as AbciEvent;
+use tendermint_rpc::abci::tag::Tag;
+use tendermint_rpc::abci::Event as AbciEvent;
```

Comment:
...but the RPC ABCI event types have migrated to `tendermint-rpc` in 0.24.

Either this change can be reverted in tendermint-rs, or we should move the event parsing infrastructure to the `relayer` crate, or rework the whole subscription mechanism for 0.24.